templates/toil-instance.yaml

```diff
@@ -151,16 +178,79 @@ Resources:
                 OWNER_EMAIL: !Ref OwnerEmail
         WriteSshFiles:
           files:
-            /home/ubuntu/.ssh/id_rsa:
+            /root/.ssh/id_rsa:
               content: !Ref RsaKey
               mode: "000400"
-              owner: "ubuntu"
-              group: "ubuntu"
-            /home/ubuntu/.ssh/id_rsa.pub:
+              owner: "root"
+              group: "root"
+            /root/.ssh/id_rsa.pub:
               content: !Ref PublicRsaKey
               mode: "000644"
+              owner: "root"
+              group: "root"
+        LaunchCluster:
+          files:
+            /lib/systemd/system/toil-cluster.service:
+              content: |
+                [Unit]
+                Description=Cluster launch/destroy scripts
+                After=network.target nss-lookup.target
+                Wants=network.target nss-lookup.target
+
+                [Service]
+                Type=oneshot
+                ExecStart=/bin/bash /home/ubuntu/create-toil-cluster.sh
+                ExecStop=/bin/bash /home/ubuntu/remove-toil-cluster.sh
+                ExecReload=/bin/bash /home/ubuntu/reload-toil-cluster.sh
+                RemainAfterExit=true
+
+                [Install]
+                WantedBy=multi-user.target
+              mode: "000644"
+              owner: "root"
+              group: "root"
+            /home/ubuntu/create-toil-cluster.sh:
```

Comment:
I think I may move these three scripts to /root/toil, since they are being run by root anyway and the ubuntu user shouldn't be executing them.